.gitlab-ci.yml

```diff
@@ -1,4 +1,4 @@
-image: python:3.9
+image: python:3.11
```

Comment:
You can nuke this `.gitlab-ci.yml` as far as I know it's not being used

Reply:
Nuked